tests/acceptance/30_custom_promise_types/08_slist_attrs.cf

```diff
@@ -0,0 +1,67 @@
+######################################################
+#
+#  Basic test of promise module using the Python library and an slist attribute
+#
+#####################################################
+body common control
+{
+        inputs => { "../default.cf.sub" };
+        bundlesequence  => { default("$(this.promise_filename)") };
+        version => "1.0";
+}
+
+#######################################################
+
+bundle common python_check
+{
+  classes:
+      "python_f_string_support" expression => returnszero("/usr/bin/python3 -c 'f\"string\"'", "useshell");
+}
+
+bundle agent init
+{
+  meta:
+      "test_skip_unsupported" string => "!python_f_string_support";
+
+  files:
+      "$(this.promise_dirname)/cfengine.py"
+        copy_from => local_cp("$(this.promise_dirname)/../../../misc/custom_promise_types/cfengine.py");
+}
+
+#######################################################
+
+promise agent multiline_insert
+{
+        interpreter => "/usr/bin/python3";
+        path => "$(this.promise_dirname)/multiline_insert.py";
+}
+
+bundle agent test
+{
+  meta:
+      "description" -> { "CFE-3444" }
+        string => "Test a simple promise module which uses the Python library and an slist attribute";
+
+  vars:
+      "lines" slist => { "hello", "from", "modules" };
+
+  multiline_insert:
+      "$(G.testfile)"
+        lines => @(lines);
```

Comment:
Humm, usually when an attribute can take a list, it can do so like `lines => { "one", "two", "three" }` or `lines => { @(lines) }`. Does this change support that?

Reply:
`lines => { "one", "two", "three" }` works just fine, I tested that first and then replaced it with this more complex case. 

Regarding `lines => { @(lines) }` -- I thought `@(list)` was actually expanded to `{ item1,...itemN }` so it doesn't make much sense to me to put extra `{ }` around it.

Reply:
> it doesn't make much sense to me to put extra `{ }` around it.

But it also just works. I guess we have some generic handling for this.